Monofoxe/Monofoxe.Engine.DesktopGL/Monofoxe - Backup.Engine.DesktopGL.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
What does this do here?

Reply:
DesktopGL is not necessarily for Windows.

Reply:
But sure, I'll add it back. I'll just remove it for myself.